contracts/utils/deploy.js

```diff
@@ -10,6 +10,7 @@ const {
   isFork,
   isRinkeby,
   isMainnetOrRinkebyOrFork,
+  isSmokeTest,
```

Comment:
We should probably enable Linting on OUSD. Stuff like missing imports would be picked up by that